examples/amp-story/player-local-stories.html

```diff
@@ -39,12 +39,12 @@ <h1>This is a NON-AMP page that embeds a story below:</h1>
     <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras viverra neque ex, sit amet varius sem maximus sed. Suspendisse potenti. Donec erat purus, sagittis sit amet tincidunt ut, maximus sit amet massa. Maecenas venenatis fringilla dui vitae vestibulum. Fusce imperdiet euismod lobortis. Nullam sagittis nunc at tristique mattis. In sodales consectetur mollis. Maecenas sollicitudin, ex vel tempor rutrum, turpis eros interdum enim, sit amet volutpat tortor dolor at ipsum. Nam posuere velit vel urna vulputate interdum. Aenean eu vulputate lorem. Praesent nec nunc sodales, egestas orci sed, hendrerit mauris. Ut blandit turpis non erat sagittis, quis fermentum odio feugiat.</div>
     <amp-story-player style="width: 360px; height: 600px;">
         <a href="./ampconf.html"
-            style="--story-player-poster: url('./img/overview.jpg');"
+            style="--story-player-poster: url('/examples/amp-story/img/overview.jpg');"
+            data-poster-portrait-src="/examples/amp-story/img/overview.jpg"
```

Comment:
The fact that the same `<a` tag has two posters is confusing. Should we move the CSS variable to the player level here and in the documentation (if needed)?